packages/zowe-explorer/src/Profiles.ts

```diff
@@ -482,10 +482,24 @@ export class Profiles extends ProfilesCache {
                             if (!Number.isNaN(Number(updValue))) {
                                 updSchemaValues[value] = Number(updValue);
                             } else {
-                                if (schema[value].optionDefinition.hasOwnProperty("defaultValue")) {
-                                    updSchemaValues[value] = schema[value].optionDefinition.defaultValue;
-                                } else {
-                                    break;
+                                switch (true) {
+                                    case schema[value].optionDefinition.hasOwnProperty("defaultValue"):
+                                        if (schema[value].optionDefinition.defaultValue === null) {
+                                            vscode.window.showInformationMessage(
+                                                localize("editConnection.rejectUnauthorize", "Operation Cancelled")
+                                            );
+                                            return undefined;
+                                        }
```

Comment:
If the user didn't enter a value for a numeric property, and the schema contains a null value, we should still create the profile without this property.

For comparison, here are the schema definitions encoding and responseTimeout
```
{
  "name": "encoding",
  "aliases": ["ec"], 
  "description": "The encoding for download and upload of z/OS data set and USS files. The default encoding if not specified is 1047.",
  "type":"number"
}
```

```
{
  "name": "responseTimeout",
  "aliases": ["rto"],
  "description": "The maximum amount of time in seconds the z/OSMF Files TSO servlet should run before returning a response. Any request exceeding this amount of time will be terminated and return an error. Allowed values: 5 - 600",
  "type": "number",
  "defaultValue": null,
  "numericValueRange": [5,600]
}
```